projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.html

```diff
@@ -115,6 +115,58 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
                 </li>
               </ng-container>
             </ul>
+            <ng-container
+              *ngIf="
+                insuredObject.childInsuredObjectList?.insuredObjects?.length > 0
+              "
+            >
+              <ng-container
+                *ngFor="
+                  let childInsuredObject of insuredObject.childInsuredObjectList
+                    .insuredObjects;
+                  let i = index
+                "
+              >
+                <ul class="accordion-list container-fluid">
+                  <li class="accordion-list-item row sub-title">
+                    <div class="col-12">
+                      {{ childInsuredObject.insuredObjectType }} {{ i + 1 }}
+                    </div>
+                  </li>
+                  <ng-container
+                    *ngIf="
+                      childInsuredObject.insuredObjectItems &&
+                      childInsuredObject.insuredObjectItems.length > 0
```

Comment:
Could you just check with one condition: childInsuredObject.insuredObjectItems**?**.length > 0

Reply:
Done!